lite/examples/image_classification/ios/README.md

```diff
@@ -37,7 +37,7 @@ Note: Objective-C developers should use the TensorFlow Lite Task Library's
 
 *   Xcode command-line tools (run `xcode-select --install`)
 
-*   [CocoaPods](https://cocoapods.org/) (run `sudo gem install cocoapods`)
+*   [CocoaPods](https://cocoapods.org/) (run `brew install cocoapods`)
```

Comment:
Could you elaborate on what the issue was?

Cocoapods's official documentation says `sudo gem install cocoapods` is the way to install CocoaPods.

Reply:
I've deleted the clone of the repo permanently, so I'm not able to access the logs. But I think it was the same thing as [this one](https://stackoverflow.com/questions/69352212/you-may-have-encountered-a-bug-in-the-ruby-interpreter-or-extension-libraries)
In my understanding, many people encountered this problem. I saw the [same question](https://developer.apple.com/forums/thread/652822) on Apple's developer forum. And someone recommends the same solution (homebrew installation) as on Stack Overflow.